qCC/ccCommandLineCommands.cpp

```diff
@@ -351,6 +352,45 @@ bool CommandChangeMeshOutputFormat::process(ccCommandLineInterface &cmd)
 	return true;
 }
 
+CommandChangeHierarchyOutputFormat::CommandChangeHierarchyOutputFormat()
+	: CommandChangeOutputFormat("Change hierarchy output format", COMMAND_HIERARCHY_EXPORT_FORMAT)
+{}
+
+bool CommandChangeHierarchyOutputFormat::process(ccCommandLineInterface& cmd)
+{
+	QString defaultExt;
+	QString fileFilter = getFileFormatFilter(cmd, defaultExt);
+	if (fileFilter.isEmpty())
+		return false;
+
+	cmd.setHierarchyExportFormat(fileFilter, defaultExt);
+	cmd.print(QObject::tr("Output export format (hierarchy) set to: %1").arg(defaultExt.toUpper()));
+
+	//look for additional parameters
+	while (!cmd.arguments().empty())
+	{
+		QString argument = cmd.arguments().front();
+
+		if (ccCommandLineInterface::IsCommand(argument, COMMAND_EXPORT_EXTENSION))
+		{
+			//local option confirmed, we can move on
+			cmd.arguments().pop_front();
+
+			if (cmd.arguments().empty())
+				return cmd.error(QObject::tr("Missing parameter: extension after '%1'").arg(COMMAND_EXPORT_EXTENSION));
```

Comment:
Style: Add "{" and "}" around single statements in an `if`

Reply:
Added troughout entire file